x-pack/solutions/security/plugins/security_solution/server/agent_builder/tools/entity_risk_score_tool.ts

```diff
@@ -219,12 +219,7 @@ export const entityRiskScoreTool = (
                       ...(score.criticality_level !== undefined && {
                         criticality_level: score.criticality_level,
                       }),
-                      ...(score.is_privileged_user !== undefined && {
-                        is_privileged_user: score.is_privileged_user,
-                      }),
-                      ...(score.privileged_user_modifier !== undefined && {
-                        privileged_user_modifier: score.privileged_user_modifier,
-                      }),
+                      ...(score.modifiers && { modifiers: score.modifiers || [] }),
```

Comment:
Hey!
No difference when working with arrays, but it looks like nullish coalescing would be more suitable? 

```suggestion
                      ...(score.modifiers && { modifiers: score.modifiers ?? [] }),
```